test/e2e/node/security_context.go

```diff
@@ -1,3 +1,5 @@
+//go:build linux
```

Comment:
this seems weird... the test is marked as `[LinuxOnly]` but can build/run from any os (the test runs as an API client)

Reply:
Makes sense, I removed the build constraint.